spec/std/crystal/hasher_spec.cr

```diff
@@ -51,6 +52,15 @@ describe "Crystal::Hasher" do
       2.hash.should eq(2_u64.hash)
     end
 
+    it "Big i64 numbers should be hashed ok" do
+      Int64::MAX.hash.should eq (Int64::MAX.hash)
```

Comment:
Please remove spaces between `eq` and the value that is expected to be _equal to_.